sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -69,6 +69,15 @@
     <Compile Include="Shared\HttpMessageSanitizer.cs" />
     <Compile Include="Shared\InitializationConstructorAttribute.cs" />
     <Compile Include="Shared\Multipart\MemoryResponse.cs" />
+    <Compile Include="Shared\MutableJsonChange.cs" />
```

Comment:
Do we actually need all of these explicitly, given the splat pattern on L52?

Reply:
The splat on L52 removes everything, so if we take these out, DynamicData will no longer be able to use them inside the Azure.Core project, unfortunately.

I was looking at this recently for something else - it looks like we do it this way because we have shared source files that don't compile in this project.  We might want to think about how to solve that at some, but I don't think it's worth blocking this PR on that work.

Reply:
Ah, that's my bad.   I missed it was a `Remove` and not `Include`